networkx/algorithms/approximation/clique.py

```diff
@@ -92,6 +92,13 @@ def clique_removal(G):
             cliques.append(c_i)
         if i_i:
             isets.append(i_i)
-
-    maxiset = max(isets)
+    
+    # max iset doesn't return the maximal set
+    # try with a clique 5 + 1 additional node, the max_clique(G) will fail and return only one node
+    # here we just return the biggest clique (iset), however, multiple biggest cliques can be found
+    lengths = [len(i) for i in isets]
+    indexOfMax = lengths.index(max(lengths))
+    maxiset = isets[indexOfMax]
```

Comment:
These three lines should simply be `maxiset = max(isets, key=len)`. Also the comment should be trimmed to 79 character lines.

Also, the comment is more like a commit note; it refers to changes introduced by the commit. Don't include it in the code.